BDArmory/Competition/BDACompetitionMode.cs

```diff
@@ -2746,9 +2745,10 @@ public void DoUpdate()
                             var surfaceAI = VesselModuleRegistry.GetModule<BDModuleSurfaceAI>(vessel);
                             if (surfaceAI != null)
                             {
-                                if ((surfaceAI.SurfaceType == AIUtils.VehicleMovementType.Land && vessel.Splashed || (vessel.horizontalSrfSpeed < surfaceAI.MaxSpeed / 10 && surfaceAI.currentStatusMode != BDModuleSurfaceAI.StatusMode.CollisionAvoidance)) //rover has gotten flipped/lost wheels and become immobilized/multiclassed to submarine
+                                if (
+                                    (surfaceAI.SurfaceType == AIUtils.VehicleMovementType.Land && (vessel.Splashed || surfaceAI.currentStatusMode == BDModuleSurfaceAI.StatusMode.Panic)) //rover has gotten flipped/lost wheels and become immobilized/multiclassed to submarine
                                     || ((surfaceAI.SurfaceType == AIUtils.VehicleMovementType.Water || surfaceAI.SurfaceType == AIUtils.VehicleMovementType.Submarine) && vessel.Landed) //vessel beached
-                                    || (surfaceAI.SurfaceType == AIUtils.VehicleMovementType.Water && vessel.IsUnderwater())) //boat sunk
+                                    || (surfaceAI.SurfaceType != AIUtils.VehicleMovementType.Submarine && vessel.IsUnderwater())) //boat sunk
```

Comment:
I'd suggest using the panic mode for the other checks too (i.e., reducing the entire check to just
```csharp
if(surfaceAI.currentStatusMode == BDModuleSurfaceAI.StatusMode.Panic)
```
), since that would consolidate all those checks in one place and the checks in `BDModuleSurfaceAI.PanicModes` are more detailed, e.g., the "sunk" condition includes checks for non-submarines that are driving on the bottom of the ocean and boats that are actively regaining the surface (though that's got a FIXME about whether it should be allowed), amongst other things.